frontend/src/post/Posts.js

```diff
@@ -22,24 +22,37 @@ class Posts extends Component {
         });
     }
 
-    renderPosts = posts => (
-        <div className='row'>
-            {posts.map((post, i) => (
-                <div className='card col-md-4' key={i}>
-                    <div className='card-body'>
-                        {/* <h5 className='card-title'>{post.title}</h5> */}
-                        <p className='card-text'>{post.body}</p>
-                        <Link
-                            to={`/posts/${post._id}`}
-                            className='btn btn-raised btn-primary btn-sm'
-                        >
-                            Read More
-                        </Link>
-                    </div>
-                </div>
-            ))}
-        </div>
-    );
+    renderPosts = posts => {
+        return (
+            <div className='row'>
+                {posts.map((post, i) => {
+                    const posterId = post.postedBy ? `/user/${post.postedBy._id}` : "" ;
+                    const posterName = post.postedBy ? post.postedBy.name : " Unknown";
+
+                    return (
+                        <div className='card col-md-4' key={i}>
+                        <div className='card-body'>
+                            <p className='card-text'>{post.body.substring(0, 100)}</p>
+                            <br/>
+                            <p className='font-italic mark'>
+                                Posted by <Link to={`${posterId}`}>{posterName}</Link>
+                            {" "}
```

Comment:
Making the writing on the (Author, Date, Link) bold makes the eyes focus on that information instead of the post.
Should we consider just italics or making the post in bold?